components/server/src/orgs/organization-service.ts

```diff
@@ -187,12 +188,31 @@ export class OrganizationService {
 
         try {
             await this.teamDB.transaction(async (db) => {
+                const members = await this.teamDB.findMembersByTeam(orgId);
+                const firstMember = members.filter((m) => m.userId !== BUILTIN_INSTLLATION_ADMIN_USER_ID).length === 0;
+                if (firstMember) {
+                    // first member (that is not an admin) is going to be an owner
+                    role = "owner";
+                    log.info({ userId: memberId }, "First member of organization, setting role to owner.");
+                }
                 await db.setTeamMemberRole(memberId, orgId, role);
                 await this.auth.addOrganizationRole(orgId, memberId, role);
```

Comment:
Unrelated to this PR, but: I just realized it seems we never take away your `owner` role here, even if we switch from `owner` to `member` :grimacing:
@svenefftinge Do we have a test case for that? :thinking:

Reply:
good catch. fixed and tested